src/Microsoft.Health.Fhir.Core/Features/Operations/Import/GetBulkImportRequestHandler.cs

```diff
@@ -0,0 +1,84 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using MediatR;
+using Microsoft.Health.Core.Features.Security.Authorization;
+using Microsoft.Health.Fhir.Core.Exceptions;
+using Microsoft.Health.Fhir.Core.Features.Security;
+using Microsoft.Health.Fhir.Core.Messages.Import;
+using Microsoft.Health.TaskManagement;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Import
+{
+    public class GetBulkImportRequestHandler : IRequestHandler<GetImportRequest, GetImportResponse>
+    {
+        private readonly ITaskManager _taskManager;
+        private readonly IAuthorizationService<DataActions> _authorizationService;
+
+        public GetBulkImportRequestHandler(ITaskManager taskManager, IAuthorizationService<DataActions> authorizationService)
+        {
+            EnsureArg.IsNotNull(taskManager, nameof(taskManager));
+            EnsureArg.IsNotNull(authorizationService, nameof(authorizationService));
+
+            _taskManager = taskManager;
+            _authorizationService = authorizationService;
+        }
+
+        public async Task<GetImportResponse> Handle(GetImportRequest request, CancellationToken cancellationToken)
+        {
+            EnsureArg.IsNotNull(request, nameof(request));
+
+            if (await _authorizationService.CheckAccess(DataActions.Import, cancellationToken) != DataActions.Import)
```

Comment:
Should only import users be able to view import job status?

Reply:
I design to allow only importer to check import status. Not sure you have any concern on that?